0.3.1

MrSerge01 opened the pull request:
okay

MrSerge01 committed:
broken ms + serverboard partial fix

ZakaHaceCosas committed:
fixed ms

ZakaHaceCosas committed:
DM owners if serverboard is broken (logChannel fails if they lack log channel)

MrSerge01 committed:
serverboard issue fixed

MrSerge01 committed:
Merge branch 'SokoraDesu:dev' into dev

MrSerge01 committed:
some more stuff

dimkauzh commented:
Let me COOK

MrSerge01 committed:
lord help me

ZakaHaceCosas committed:
fix topgg reminders + make commands ephemeral where it makes sense to

MrSerge01 committed:
I might rip my hair in this timeline

MrSerge01 committed:
safeChannel

MrSerge01 committed:
quick

MrSerge01 committed:
no more newline

MrSerge01 committed:
Merge branch 'SokoraDesu:dev' into dev

ZakaHaceCosas reviewed (approved):
LGTM

dimkauzh reviewed (approved):
Awesome, can't wait to test this with Docker on production. LGTM!

Golem642 commented on src/commands/about.ts:
```diff
@@ -67,5 +67,5 @@ export async function run(interaction: ChatInputCommandInteraction) {
   );
 
-  await interaction.reply({ embeds: [embed], components: [row] });
+  await interaction.reply({ embeds: [embed], components: [row], flags: "Ephemeral" });
```
Meh, we can leave this one as a normal message

MrSerge01 replied:
valid, someone else can clickt he vote link and we go 🤑🤑🤑🤑🤑

Golem642 commented on src/utils/database/settings.ts:
```diff
@@ -281,6 +281,8 @@ export async function getSetting<
       case "INTEGER":
         return parseInt(value) as SqlType<typeof set.type>;
+      case "CHANNEL":
+        return value.replace("<#", "").replace(">", "");
```
We're going to need a perma fix in the form of a db migration for this instead of just avoiding the problem

MrSerge01 replied:
maybe just check for if the value has <# and > then it setSetting()s it to a value that's good for use :D
oh and i still gotta find a way to clean the other settings that are just words lmao

Golem642 reviewed (approved):
Honestly if the bugs encountered are fixed and there are no regression with things like ms or newline then yeah this is good 👍

ZakaHaceCosas committed:
fix DB inconsistencies (requires migration)
removes the 'fix' to some IDs being stored as <#> too, as this requires the DB to be manually migrated anyway to prevent data loss
